packages/toolkit/src/components/NewsLetterCard.tsx

```diff
@@ -0,0 +1,122 @@
+"use client";
+import { Icons } from "@instill-ai/design-system";
+import { ImageWithFallback } from "./ImageWithFallback";
+import { useState } from "react";
+import { useBlogPosts } from "../lib/react-query-service/misc/useBlogPosts";
+
+interface BlogPostData {
+  id: string;
+  imageUrl: string;
+  title: string;
+  publishedOn: string;
+  themeImgAlt: string;
+  themeImgSrc: string;
+  slug: string;
+}
+
+
+
+const NewsLetterCard = () => {
+  const { data: blogPosts = [], isLoading, isError } = useBlogPosts();
+
+  const [currentIndex, setCurrentIndex] = useState(0);
+
+  const NewsLetterCardSkeleton = () => {
+    return (
+      <div className="flex w-full flex-col px-2">
+        <div className="h-[250px] w-full animate-pulse bg-semantic-bg-secondary" />
+        <div className="mb-2 h-5 w-1/2 animate-pulse rounded bg-semantic-bg-secondary" />
+      </div>
+    );
+  };
+
+  const handlePrevious = () => {
+    setCurrentIndex((prevIndex) => (prevIndex === 0 ? 0 : prevIndex - 1));
+  };
+
+  const handleNext = () => {
+    setCurrentIndex((prevIndex) =>
+      prevIndex === blogPosts.length - 1 ? prevIndex : prevIndex + 1
+    );
+  };
+
+  if (isLoading) {
+    return <NewsLetterCardSkeleton />;
+  }
+
+  if (isError) {
+    return <div>Error fetching blog posts.</div>;
+  }
+
+  const { imageUrl, title, publishedOn, themeImgAlt, slug } =
+    blogPosts[currentIndex];
+
+  return (
+    <div className="flex h-[450px] flex-col gap-y-2 rounded-sm border border-semantic-bg-line p-4">
+      <h2 className="mb-4 text-2xl font-bold">What's New?</h2>
+      <div className="relative h-[250px] w-full">
+        <a
+          href={`https://www.instill.tech/blog/${slug}`}
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          <ImageWithFallback
+            src={imageUrl}
+            width={600}
+            height={400}
+            alt={themeImgAlt}
+            fallbackImg={
+              <Icons.Box className="h-8 w-8 stroke-semantic-fg-primary" />
+            }
+            className="h-full w-full object-cover"
+          />
+        </a>
+      </div>
+      <button
+        type="button"
+        className="my-1 w-min whitespace-nowrap rounded-sm bg-semantic-accent-bg px-2 py-2 capitalize text-semantic-accent-default product-button-button-2 hover:!underline "
+      >
+        {publishedOn}
+      </button>
```

Comment:
@thewbuk Although the design-handover have this as button, but since it didn't has any button functionality, we can replace the button element with div and utilize the button style like this 

```
<div className={cn(buttonVariants({ variant: "secondaryColour", size: "sm" }))}> 
```

A side note, when it comes to button, in the desgin-handover it will reference its variant, and we have a unified button in the design-system, please use that component first 🦤